tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>What does the load function do? What is the "necessary format"? </p><p> </p><p>(I am just asking the naive questions that a newbie might have :) )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>